karma.conf.js

```diff
@@ -16,6 +22,7 @@ module.exports = function(config) {
     // list of files / patterns to load in the browser
     files: [
       'node_modules/es6-promise/dist/es6-promise.js',
+      'node_modules/babel-polyfill/dist/polyfill.js',
```

Comment:
does this make the `es6-promise` line above redundant? `core-js` includes ES6 promises: https://github.com/zloirock/core-js#ecmascript-6-promise